roles/sql-command/tasks/create-user.yml

```diff
@@ -68,4 +68,4 @@
     - name: Remove my.cnf file
       include: remove-my-cnf.yml
 
-  when: "{{ rds_dbs | selectattr('db_engine', 'match', 'MySql')| list |count > 0 }}"
+  when: rds_dbs | selectattr('db_engine', 'match', 'MySql')| list |count > 0
```

Comment:
L71: is this working? , multiple jinja filters on a single line always give me upset stomachs, see https://github.com/ansible/ansible/issues/17773#issuecomment-249963023